src/CMakeLists.txt

```diff
@@ -27,40 +49,105 @@ if(WIN32)
 
     set(RADARE2_INCLUDE_DIRS "${IAITO_WIN32_DIR}/radare2/include/libr" "${IAITO_WIN32_DIR}/include")
 endif()
+#
+# Win32 setting ends
 
-
-find_package(Radare2 REQUIRED)
-include_directories(${RADARE2_INCLUDE_DIRS})
-link_directories(${RADARE2_LIBRARY_DIRS})
-
-
-if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
-        OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    add_definitions(-Wall -Wextra)
+# Apple
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+	set(CMAKE_CXX_FLAGS "-mmacosx-version-min=10.7 -std=gnu0x -stdlib=libc++")
 endif()
 
 
-# Parse iaito.pro to get filenames
-include(QMakeProParse)
-parse_qmake_pro("${CMAKE_CURRENT_SOURCE_DIR}/iaito.pro" IAITO_PRO)
-set(SOURCE_FILES ${IAITO_PRO_SOURCES})
-set(HEADER_FILES ${IAITO_PRO_HEADERS})
-set(UI_FILES ${IAITO_PRO_FORMS})
-set(QRC_FILES ${IAITO_PRO_RESOURCES})
-
-message(STATUS "sources from iaito.pro: ${SOURCE_FILES}")
-message(STATUS "headers from iaito.pro: ${HEADER_FILES}")
-message(STATUS "forms from iaito.pro: ${UI_FILES}")
-message(STATUS "resources from iaito.pro: ${QRC_FILES}")
-
+# Compiler settings
+#
+if(CMAKE_CXX_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_IS_CLANGXX)
+	set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -Wextra")
+endif(CMAKE_CXX_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_IS_CLANGXX)
```

Comment:
set(CMAKE_CXX_STANDARD 11) and -std=c++11 are same afaik.
well CMAKE_CXX_COMPILER_IS_GNUCXX do exist https://cmake.org/cmake/help/latest/variable/CMAKE_COMPILER_IS_GNUCXX.html.

CMAKE_COMPILER_IS_CLANGCXX do seem to exist but I also see typo in my PR. But I am OK if it isn't required